test/core/messages.integration.test.ts

```diff
@@ -141,6 +141,72 @@ describe('messages integration', () => {
     ]);
   });
 
+  it<TestContext>('should be able to update and receive update messages', async (context) => {
```

Comment:
Could we also have a test that gets history with updates? We can just skip it for now, but that way we don't forget about it. I've done the same below for deletes I think

Reply:
Added